src/org/zaproxy/zap/extension/autoupdate/ManageAddOnsDialog.java

```diff
@@ -254,14 +259,17 @@ private JPanel getInstalledAddOnsPanel() {
 							java.awt.Color.black));
 
 			getInstalledAddOnsTable();
-			
+

```

Comment:
Unrelated indentation changes should not be done in the same commit. [1]

[1] https://github.com/zaproxy/zaproxy/wiki/DevGuidelines